source/sha_override.c

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <ec_utils.h>
+#include <openssl/sha.h>
+
+#define INIT_DATA_h0 0x67452301UL
```

Comment:
Are these magic constants (here and in the `_Init` functions below) specified in some standard for SHA?

I am not super familiar with crypto implementations, but if you could consider leaving a comment pointing to the standard somewhere at the top of this doc. Something similar to the comment you left in the `md5_override.c` file would be nice.

Reply:
I added a comment and a link to the corresponding file in the openssl repo.